basic/01a_lookup.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Initialize the lookup of modules and display information for any modules
+ * on the network. A 'module' can be an actuator, and I/O board, or a 
+ * mobile device running the Mobile I/O app.
+ *
+ * For more information, go to http://docs.hebi.us/tools.html#cpp-api
+ *
+ * HEBI Robotics
+ * September 2018
+ */
+
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <string>
+
+#include "lookup.hpp"
+
+using namespace hebi;
+
+int main() {
+  // Create a Lookup object
+  Lookup lookup;
+
+  // Wait 2 seconds for the module list to populate, and then print out its contents
+  std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+
+  std::cout << std::endl;
+  auto entry_list = lookup.getEntryList();
+  for (auto entry : *entry_list)
+    std::cout
+      << "Name: " << entry.name_ << std::endl
+      << "Family: " << entry.family_ << std::endl << std::endl;
```

Comment:
Using `std::endl` is generally bad practice, and I'd like to propagate better practices to our users

Reply:
[This](https://github.com/isocpp/CppCoreGuidelines/issues/357#issuecomment-153515258) is enough of an argument not to use it